helm-chart/splunk-operator/values.yaml

```diff
@@ -0,0 +1,137 @@
+# Splunk image
+image:
+  repository: docker.io/splunk/splunk:edge
+
+# The kube-rbac-proxy is a small HTTP proxy for a single upstream, that can perform RBAC 
+# authorization against the Kubernetes API.
+# reference: https://github.com/brancz/kube-rbac-proxy
+kubeRbacProxy:
+
+  # Specify kube-rbac-proxy image
+  image: 
+    repository: gcr.io/kubebuilder/kube-rbac-proxy
+    pullPolicy: IfNotPresent
+    tag: "v0.11.0"
+
+  # Set resource requests and limits for kube-rbac-proxy container
+  resources:
+    limits:
+      cpu: 200m
+      memory: 100Mi
+    requests:
+      cpu: 100m
+      memory: 20Mi
+
+# Splunk Operator configurations
+splunkOperator: 
+
+  # Enable to deploy Splunk Operator
+  enabled: true
```

Comment:
since this is mandatory should we not check if operator is not installed, then install it

Reply:
Removed this in favor of a condition within Chart.yaml. The operator is a dependency to the enterprise chart and installs with the enterprise chart. If a user decides they would like to install the enterprise without installing the operator and related resources they can change the condition through the CLI.